components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.scss

```diff
@@ -0,0 +1,158 @@
+@import "@carbon/grid/scss/grid";
+
+:host {
+  @extend .bx--grid;
+  position: absolute;
+  top: calc(100% + 8px);
+  right: 0;
+  padding: 16px;
+  border-radius: 8px; // not variable
+  background: $chef-white;
+  box-shadow: 8px 8px 24px 0 rgba(197, 199, 209, 0.6);
+  width: 100%;
+  transform-origin: top right;
+
+  @include carbon--breakpoint-up(md) {
+    max-width: 467px;
+  }
+
+
+  * {
+    font-family: 'Inter Regular';
+  }
+
+  .bx--row {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .instruction {
+    margin: 0;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    list-style-type: none;
+    padding: 0;
+    margin-top: 15px;
+    margin-bottom: 0;
+    min-height: 420px;
+    @include carbon--breakpoint-up(md) {
+      max-height: 392px;
+    }
+
+
+    .filter-button-container {
+      width: 100%;
+      padding-left: 0;
+      padding-right: 0;
+
+      @include carbon--breakpoint-up(md) {
+        width: 50%;
+
+        &:nth-child(-n + 10) {
+          padding-right: 8px;
+        }
+      }
+
+
+
+    }
+
+    .filter-button {
+      margin-bottom: 8px;
+      padding: 0;
+      border: 1px solid $chef-desktop-light;
+      border-radius: $chef-border-radius;
+      overflow: hidden;
+      cursor: pointer;
+
+      .check {
+        visibility: hidden;
+        // temp color
+        color: #4535D7;
+      }
+
+      span {
+        display: flex;
+        justify-content: space-between;
+        padding: 7px 16px 5px;
+        width: 100%;
+        height: 100%;
+        background: $chef-white;
+        font-size: 14px;
+        // not yet named
+        color: #394261;
+        letter-spacing: .25px;
+        pointer-events: none;
+      }
+
+      // selected styles
+      &.selected {
+        border: 1px solid transparent;
+        background-image: $chef-purple-gradient;
+
+        .check {
+          visibility: visible;
+        }
+      }
+    }
+  }
+
+  .action-button-row {
+    @extend .bx--row;
+    justify-content: space-between;
+    margin-top: 8px;
+  }
+
+
+  // checkbox overwrites
+  ::ng-deep chef-checkbox {
+    margin-top: 0;
+    margin-bottom: 0;
+
+    &[aria-checked=true] .check-wrap {
+      // not yet named colors
+      border-color: #4535D7;
+      background-color: #4535D7;
+    }
+
+    &[aria-checked=false] .check-wrap {
+      border: 1px solid $chef-desktop-light;
+    }
+
+    .label-wrap {
+      margin-left: 8px;
+      color: #394261;
+      letter-spacing: .25px;
+    }
+  }
+
+  ::ng-deep chef-button {
+    color: #2E1EC1;
+    border-radius: 8px;
+    font-weight: bold;
+    margin-top: 0;
+    margin-bottom: 0;
+
+    &[primary] {
+      background: #DFDCFF;
+      border: none;
+      box-shadow: 2px 2px 4px 0 #B3ABFB, -2px -2px 12px 0 #ECF0F9;
+      margin-right: 0;
+    }
+
+    &[disabled] {
+      // temp until get styles
+      background: desaturate(#DFDCFF, 80%);
+    }
+
+    button {
+      padding: 0 16px;
+    }
+  }
```

Comment:
Seems like we should define a lot of this in once place and then swap in this styling across the Desktop view (and whole app?) all at once. Bunch of mixed colors and fonts on master isn't the greatest look; but this is all still contained in an unshipped WIP feature so doesn't matter I suppose. I'm just concerned about the possibility of field teams getting their hands on this feature (when?) while we are still working out the split-visual identity of our UI.

At this point I think I'm the black sheep on this topic, which isn't even specific to this PR, so y'all please feel free to ignore my ramblings today 😄

Reply:
believe it or not, I agree with you on this.  I guess with Chef Conf coming up I felt rushed to get everything in.  Maybe i'll undo them for now - lets chat about it next standup